### ISSUE TITLE
dcr: inconsistent wallet address

### ISSUE THREAD

**Issue by ukane-philemon** [decred/dcrdex#1513]
Steps to reproduce: 
1. Check wallet address with `dcrctl --testnet --walllet getaccountaddress name`
2. Start dexc which will display the dcr wallet address from `step 1`.
3. Check the dcr wallet address again with `dcrctl --testnet --walllet getaccountaddress name`, this will return a new address
4. Refresh dexc client wallet page, no changes made to dcr wallet address.
5. click the `new address` button on dcr wallet deposit page, this returns the address from `step 3`.
6. Repeat `step 3` and a new address is returned. 
7. Repeat `step 5` and the wallet address from `step 6` is returned.

generating new dcr wallet address with the `new deposit address` button is not in sync with `dcrctl --testnet --wallet getaccountaddress name`.

tl;dr: `dcrctl --testnet --wallet getaccountaddress name` is one new address ahead of dexc. Is this an expected behaviour?

**Comment by chappjc** [comment]
Deposit address is working as intended.

getaccountaddress is deprecated also.

```
getaccountaddress "account"

DEPRECATED -- Returns the most recent external payment address for an account that has not been seen publicly.
A new address is generated for the account if the most recently generated address has been seen on the blockchain or in mempool.
```

**Comment by ukane-philemon** [comment]
@chappjc thanks for your feedback. Closing....
